JS/app.js

```diff
@@ -25,19 +26,49 @@ photoArray.push(new Photo('img/kitten12.jpg'));
 photoArray.push(new Photo('img/kitten13.jpg'));
 photoArray.push(new Photo('img/kitten14.jpg'));
 
-//etc
+var leftVotes;
+var rightVotes;
 
-var Tracker = function() {
-this.leftPhoto = this.getRandomInt();
-this.rightPhoto = this.getRandomInt();
+var ctx = document.getElementById("myChart").getContext("2d");
+console.dir(Chart)
+var myNewChart = new Chart(ctx).Doughnut([{value: leftVotes, color: 'red'}, {value: rightVotes, color: 'green'}]);
+console.dir(Chart);
 
+
+var Tracker = function() {
+this.leftPhoto = '';
+this.rightPhoto = '';
 };
+var tracker = new Tracker();
 
 Photo.prototype.highlight = function() {
 //highlight the photo after it is clicked
+//Does this need JS functionality? Can it be done with just css????
```

Comment:
yes, this can be done specifically with CSS, though you can do it with jQuery as well.